max_speed: 2.5 # km/h
max_steering: 20.0 # degree
